requirements-dev.txt

```diff
@@ -3,7 +3,7 @@ cmake
 scikit-build
 cython
 asv
-cvxopt
+cvxpy
```

Comment:
This should be `cvxpy>=1.0.0` in case someone is using an old version.

Reply:
Done in: https://github.com/Qiskit/qiskit-aer/pull/430/commits/426bcfccc69684c4ba50848b4ec8605ca59ad78b